README.md

```diff
@@ -10,10 +10,21 @@ Downloads the following materials:
 * Pages and Labels: Embedded Opencast and Youtube Videos
 
 # Setup
-Initially you need to install the requirements (**Python version >=3.6 is needed.**):
+This software requires **Python version >=3.6**.
+
+Obtain the source, for example using `git clone` or downloading and unzipping a zip. The following exemplary commands are to be used from inside that downloaded `syncMyMoodle` directory.
+
+Since you need to install dependencies to run syncMyMoodle, the recommended way to install them is by first creating and activating a virtual environment ([more info](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment)):
```

Comment:
I think this sentence (and the previous) could be shortened a bit

Reply:
I updated, please mark as resolved or offer an alternative?